test/correctness/simd_op_check.cpp

```diff
@@ -195,6 +195,7 @@ struct Test {
         for (auto p : image_params) {
             p.reset();
             p.set_host_alignment(128);
+            p.dim(0).set_min(0);
```

Comment:
Why is this additional constraint necessary?

Reply:
This additional constraint is necessary for testing 'valign' that relies on Halide being able to figure out the alignment of addresses in loads/stores. Without setting the min to a value like this, Halide cannot figure out the alignment of loads and stores.